requirements.txt

```diff
@@ -2,7 +2,7 @@
 nose2>=0.9.2
 nose2[coverage_plugin]>=0.6.5
 pylint>=2.6.0
-cryptography~=3.3.1
+cryptography~=3.3.2
```

Comment:
this was changed?

Reply:
Because of the ~ it takes 3.3.2 anyway even if 3.3.1 written. So its better to change it here to be clear.
@sashaCher said 3.3.2 has security patch